src/run.ts

```diff
@@ -228,36 +228,48 @@ yargs(process.argv.slice(2))
   )
   .command(
     "base-update",
-    "this will upgrade your code to the latest version of the base template",
+    "this will update your code to the latest version of the base template",
     {},
     async () => {
-      const addRemoteCommand = [
-        "git",
-        "remote",
-        "add",
-        "base",
-        "https://github.com/Enterprise-CMCS/macpro-base-template",
-      ];
-
-      await runner.run_command_and_output(
-        "Upgrade from Base | adding remote",
-        addRemoteCommand,
+      const checkForBaseRemoteCommand = ["git", "remote", "show", "base"];
```

Comment:
I’m reconsidering this.  Now that I added the logic to selectively silence output, it may be cleaner to just silence the add base command.  Thoughts?